payjoin/src/send/v2/mod.rs

```diff
@@ -236,6 +242,7 @@ impl Sender {
         self.v1.endpoint.receiver_pubkey()
     }
 
+    /// The payjoin endpoint to send the request to
     pub fn endpoint(&self) -> &Url { self.v1.endpoint() }
```

Comment:
As described in the earlier comment, this definitely isn't the endpoint the request gets sent to. And so I request the docstring is changed to `The endpoint in the Payjoin URI` or similar. I don't want this to imply one should send a request here.

In fact, in v2 if you sent the request here and parsed in a v2 `pj=` param, you'd be directly connecting to the payjoin directory which would leak metadata. If the endpoint were a v1 param it'd be fine.